spec/models/entity_spec.rb

```diff
@@ -56,5 +56,13 @@
     end
   end
 
+  context 'searching by name works fine' do
+    it "finds an entity even if partial name of the entity is supplied" do
+      entity = create(:entity)
```

Comment:
This should explicitly specify that the name includes "name", so it doesn't break if the factory changes. (And it should also have a name where "name" is a subset, so that we're definitely testing that it finds a partial match.)

The call to `.save` is unneeded as `create` builds and saves. (`build` just creates, and hence requires a `save`.)